interfaces/builtin/hardware_observe.go

```diff
@@ -29,12 +29,19 @@ const hardwareObserveConnectedPlugAppArmor = `
 # from the system.  this is reserved because it allows reading potentially sensitive information.
 # Usage: reserved
 
+# used by lscpu
```

Comment:
Changes in this file look a bit unreleated, but they do look fine and its not enough to justify a separate branch I think.